v23.1/create-changefeed.md

```diff
@@ -268,7 +270,11 @@ For step-by-step guidance on creating a changefeed connected to Kafka, see the [
 (1 row)
 ~~~
 
-For more information on how to create a changefeed that emits an [Avro](https://avro.apache.org/docs/1.8.2/spec.html) record, see [this step-by-step example](changefeed-examples.html#create-a-changefeed-connected-to-kafka-using-avro). The parameters table on the [Changefeed Sinks](changefeed-sinks.html#kafka-parameters) page provides a list of all kafka-specific query parameters.
+For more information, see:
+
+- [This step-by-step example](changefeed-examples.html#create-a-changefeed-connected-to-kafka-using-avro) to create a changefeed that emits an [Avro](https://avro.apache.org/docs/1.8.2/spec.html) record.
```

Comment:
nit: `[This step-by-step example]` --> `The [Create a changefeed connected to Kafka] tutorial to create a ...` to match the other two bullet points.

Reply:
Done!

Reply:
Done! Thanks!